assets/sass/04-elements/links.scss

```diff
@@ -16,7 +16,7 @@ a:hover {
 }
 
 .site a:focus {
-	outline: 2px solid var(--wp--style--color--link, var(--global--color-primary));
+	//outline: 1px solid var(--wp--style--color--link, var(--global--color-primary));
```

Comment:
(This is just a test to allow people to try the default to see which would work best.)